Sources/Sentry/Public/SentrySpanProtocol.h

```diff
@@ -10,9 +10,40 @@ NS_SWIFT_NAME(Span)
 @protocol SentrySpan <SentrySerializable>
 
 /**
- * The context information of the span.
+ * Determines which trace the Span belongs to.
  */
-@property (nonatomic, readonly) SentrySpanContext *context;
+@property (nonatomic) SentryId *traceId;
+
+/**
+ * Span id.
+ */
+@property (nonatomic) SentrySpanId *spanId;
+
+/**
+ * Id of a parent span.
+ */
+@property (nullable, nonatomic) SentrySpanId *parentSpanId;
+
+/**
+ * If trace is sampled.
```

Comment:
`m`: The sample decision has 3 states.
```suggestion
 * The sampling decision of the trace.
```